docs/git-node.md

```diff
@@ -336,7 +338,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
+Update or patch the V8 engine.
```

Comment:
Unrelated to this PR, and likely undesirable (the two spaces forces the markdown rendering tools to include a line break)
```suggestion
Update or patch the V8 engine.  
```